Sources/StreamChat/Repositories/OfflineRequestsRepository.swift

```diff
@@ -23,6 +23,10 @@ extension Endpoint {
 /// OfflineRequestsRepository handles both the enqueuing and the execution of offline requests when needed.
 /// When running the queued requests, it basically passes the requests on to the APIClient, and waits for its result.
 class OfflineRequestsRepository {
+    enum Constants {
+        static let retryHoursThreshold: CGFloat = 12
+        static let secondsInHour: CGFloat = 3600
+    }
```

Comment:
Could we move the `retryHoursThreshold` to the actual repository and maybe make it configurable in the `ChatClientConfig`?

Reply:
done